### DISCUSSION TITLE
I can not find Morocoo in SMS authentication country list

### DISCUSSION THREAD

**Discussion by dragonsxslayer** [community/community#66497] [Other Feature Feedback, Questions, & Ideas]
I'm trying to enable 2fa using SMS authentication but i can not find Morocco in country list  why :(
(prefix: +212)

**Comment by queenofcorgis** [comment]
If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We update this list periodically.


You can set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

**Reply by dragonsxslayer** [reply]
i usually use sms way but it's really sad to know my country isn't listed , this issue been since 2020 i guess

**Comment by zakaria-chahboun** [comment]
@queenofcorgis 
I find it quite unusual that I can't locate Morocco in the SMS authentication list.

**Reply by queenofcorgis** [reply]
@zakaria-chahboun as I mentioned above this means at this time we cannot reliably deliver texts messages to your country, but we do revisit the list on a regular visit. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA. 

Regardless of country, we recommend you use a TOTP app instead, for more information on how to set that up, review our documentation [Configuring Two Factor Authentication Using a TOTP app](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

**Reply by SaadM33** [reply]
hello  @queenofcorgis  , i too couldnt find morocco when doing 2fa , so i tried TOTP , but when i try to submit an appeal form , they only prompt me with sms 2fa with no option for TOTP

**Comment by InfoXMax** [comment]
Hello ,

Is there any update regarding this ? every website on the planet is fine with morocco number except github ? while pushing us to use sms or Oauth , i don't wanna use any app so i decide to use sms but still didn't fixed for morocco ???

**Reply by ghostinhershell** [reply]
Hi @InfoXMax 👋🏾 

Thank so much for writing in. If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

You can also find this, and other frequently asked questions, in our [Two Factor Authentication Frequently Asked Questions 🔑 🗝️](https://github.com/orgs/community/discussions/68240) post.

Let me know if you have any additional questions. Thanks!

**Comment by elqessouartariq** [comment]
Hello ,
I see that it's been 2y and we still can't use 2FA sms  for Moroccan numbers (+212)
